dubbo-configcenter/dubbo-configcenter-zookeeper/src/main/java/org/apache/dubbo/configcenter/support/zookeeper/ZookeeperDynamicConfiguration.java

```diff
@@ -98,57 +94,47 @@ protected void initWith(URL url) {
         }
     }
 
+    /**
+     * @param key e.g., {service}.configurators, {service}.tagrouters, {group}.dubbo.properties
+     * @return
+     */
     @Override
-    protected String getTargetConfig(String key, String group, long timeout) {
-        // when group is not null, we are getting startup configs from Config Center
-        // for example, group=dubbo, key=dubbo.properties
-        if (StringUtils.isNotEmpty(group)) {
-            key = group + "/" + key;
-        }
-        // when group is null, we are fetching governance rules.
-        // for example, key=org.apache.dubbo.DemoService.configurators
-        else {
-            int i = key.lastIndexOf(".");
-            key = key.substring(0, i) + "/" + key.substring(i + 1);
+    public Object getInternalProperty(String key) {
+        ChildData childData = treeCache.getCurrentData(key);
+        if (childData != null) {
```

Comment:
can be simplified with 
`return childData!=null ?return new String(childData.getData(), StandardCharsets.UTF_8) :null;`